### PR TITLE
HIPP-1758: Add delete credential back to the environment page

### PR DESCRIPTION
**Pull request by victorarbuesmallada**
This PR is pretty much bringing back this [other one](https://github.com/hmrc/api-hub-frontend/commit/8d1355a0a3f00ee360efc1b6d5dbf733db8d1b82).

### CODE REVIEW

File: conf/application.routes [hunk 1]
```diff
@@ -1,79 +1,81 @@
...
-GET         /environments/:id/:environment                              controllers.application.EnvironmentsController.onPageLoad(id: String, environment: String)
+GET         /environment-and-credentials/:id                                        controllers.application.EnvironmentAndCredentialsController.onPageLoad(id: String)
+GET         /environments/:id/:environment                                          controllers.application.EnvironmentsController.onPageLoad(id: String, environment: String)
 
-GET         /add-credential-checklist/:id                               controllers.application.AddCredentialController.checklist(id: String)
-POST        /add-credential-success/:id                                 controllers.application.AddCredentialController.addProductionCredential(id: String)
-POST        /add-development-credential/:id                             controllers.application.AddCredentialController.addDevelopmentCredential(id: String)
+GET         /delete-credentials/:id/client/:clientId/environment/:environmentId     controllers.application.EnvironmentsController.onDeleteCredential(id: String, clientId: String, environmentId: String)
```
Comment (victorarbuesmallada) [inline conf/application.routes:10]: happy to change this into a post and have [this](https://github.com/hmrc/api-hub-frontend/pull/481/files#diff-f8c657143410bd53e18783c3f31252131294d7d3b1cc076f2152f6ddfa587bb9R89-R95) wrapped in a form.

File: app/views/application/EnvironmentsView.scala.html [hunk 2]
```diff
@@ -44,6 +44,12 @@ <h4 class="govuk-!-margin-0">
     </div>
 }
 
+@canDeleteCredentials = @{
+    credentials.size > 1 &&
```
[the remark this answers is not in the document]
Comment (victorarbuesmallada) [reply app/views/application/EnvironmentsView.scala.html:48]: we had a call and we've agreed that the logic is different and that this is fine as it is

**Review by rdawson-hmrc — DISMISSED** [review]

**Review by PaulCDurham — CHANGES_REQUESTED** [review]
Main issue is allowing support users without SC to delete a production credential.

File: conf/application.routes (hunk 1 above)
Comment (PaulCDurham) [inline conf/application.routes:10]: Please change to a POST.

**Review by PaulCDurham — CHANGES_REQUESTED** [review]

File: conf/application.routes (hunk 1 above)
Comment (victorarbuesmallada) [reply conf/application.routes]: Needed to tabulate the rest of the endpoints

**Review by PaulCDurham — APPROVED** [review]

**Pull request merged**